modules/terraform-aws-alternat/lambda.tf

```diff
@@ -8,6 +8,7 @@ resource "aws_lambda_function" "alternat_autoscaling_hook" {
   environment {
     variables = {
       PRIVATE_SUBNET_SUFFIX = var.subnet_suffix
+      CHECK_URLS            = var.connectivity_test_check_urls
```

Comment:
Does the `list(string)` get serialized to a `string` in some way that's consumable by the Lambda?

Reply:
Oh, good catch, this may be broken...